File: pertemuan2/diskon/diskon1/diskon1.js
let harga1 = 25000,
  harga2 = 15000,
  harga3 = 14000;
let total = harga1 + harga2 + harga3;
let diskon = total * 0.05;
console.log(`Total setelah diskon 5%: ${(total - diskon).toFixed(2)}`);

File: pertemuan2/diskon/diskon1/diskon1.c
#include <stdio.h>

int main() {
    int harga1 = 25000, harga2 = 15000, harga3 = 14000;
    int total = harga1 + harga2 + harga3;
    float diskon = total * 0.05;
    float total_diskon = total - diskon;
    printf("Total setelah diskon 5%%: %.2f\n", total_diskon);
    return 0;
}